Tools/clinic/clinic.py

```diff
@@ -2593,11 +2593,10 @@ def __repr__(self) -> str:
 def parse_file(
         filename: str,
         *,
-        ns: argparse.Namespace,
         output: str | None = None,
+        verify: bool = True,
+        limited_capi: bool = DEFAULT_LIMITED_CAPI,
```

Comment:
If we are using a global `DEFAULT_LIMITED_CAPI` constant for the default value here, then I think we should also use it for the default value in `Clinic.__init__`, instead of hardcoding it to `False` there:

https://github.com/python/cpython/blob/0e8b3fc718c8a1c4de558c553d9e05049c1dbec6/Tools/clinic/clinic.py#L2393

Though, I'm actually not sure either `verify` or `limited_capi` need to have default values. There aren't any places in `clinic.py` where `parse_file` is currently called without explicitly passing values to these parameters. And in general, I prefer to avoid having default values unless there's a good reason to have a default value. It forces you to be explicit about the values that are being passed to the function at every callsite, which reduces the potential for accidental bugs to creep in.

Reply:
I added a variable for the default to consider changing it tomorrow if AC is shipped as a third party tool.

Reply:
> If we are using a global DEFAULT_LIMITED_CAPI constant for the default value here, then I think we should also use it for the default value in Clinic.__init__, instead of hardcoding it to False there: (...)

Done.

If you prefer, I can just remove DEFAULT_LIMITED_CAPI. Changing the default tomorrow is not a big deal.